src/main/java/com/google/sps/BlobUtils.java

```diff
@@ -127,11 +128,12 @@ public static Filter combinedGetFilters(HttpServletRequest request,
   }
 
   /**
-   * Returns either the blobkey string for storage in the database or null.
+   * Returns either the blobkey string and file extension for storage in the
+   * database or null.
    * @param     {HttpServletRequest}    request
-   * @return    {String}
+   * @return    {ArrayList<String>}
```

Comment:
When returning multiple values, it's better to make a little class to hold the values, because an ArrayList<string> is very general and doesn't tell me anything about how many values there are or what the values mean.

This may be a bigger change, so if you want to leave a TODO for now and deal with it later, that's OK.